public/app/plugins/datasource/cloud-monitoring/dataquery.gen.ts

```diff
@@ -10,7 +10,7 @@
 
 import * as common from '@grafana/schema';
 
-export const DataQueryModelVersion = Object.freeze([0, 0]);
+export const pluginVersion = "1.0.0";
```

Comment:
think sth went wrong here with the version generation?

Reply:
I copy [this](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/state/PanelModel.ts#L699-L701) logic. Its happening because `plugin.json` has the version set.

I can change to use only Grafana version for everything if you prefer.

Reply:
Ok, makes sense!